### ISSUE TITLE
need Help to Install additional drivers

### ISSUE THREAD

**Issue by pcfreaxx** [ChimeraOS/chimeraos#531]
Hi. In our Forum we wanna try to run Chimera on PS4. It's still an idea yet. But in past there were many Arch Linux on ps4linux. I wanna try these drivers https://github.com/whitehax0r/ArchLinux-PS4-Drivers.

For the first steps i need to Install wget, git etc. but pacman seems not working.

**Comment by alkazar** [comment]
The system is immutable. But you can unlock it (until next update) with:
`sudo frzr-unlock`

That is good for testing things. Once you have it figured out, you can create your own fork with the packages you need by following these instructions:
https://github.com/ChimeraOS/chimeraos/wiki/Development#forking

**Comment by ruineka** [comment]
> Hi. In our Forum we wanna try to run Chimera on PS4. It's still an idea yet. But in past there were many Arch Linux on ps4linux. I wanna try these drivers https://github.com/whitehax0r/ArchLinux-PS4-Drivers.
> 
> For the first steps i need to Install wget, git etc. but pacman seems not working.

Keep me posted on your progress. I'm rather interested in seeing if the community can get ChimeraOS running on the PS4.

**Comment by pcfreaxx** [comment]
Yeah, sure. First i have to battle with llvm-14.06. I will update you.
